litmus-portal/frontend/src/views/ChaosWorkflows/BrowseAnalytics/WorkflowComparisonTable/TableData.tsx

```diff
@@ -53,16 +54,18 @@ const TableData: React.FC<TableDataProps> = ({
         </Typography>
       </TableCell>
       <TableCell>
-        &nbsp;
-        <img
-          src="/icons/calender.svg"
-          alt="Calender"
-          className={classes.calIcon}
-        />
-        <Typography className={classes.tableObjectRegularity}>
-          {/* data.regularity */}
-          {t('chaosWorkflows.browseAnalytics.workFlowComparisonTable.once')}
-        </Typography>
+        <div className={classes.regularityData}>
+          <div className={classes.iconDiv}>
+            <img src="/icons/calender.svg" alt="Calender" />
+            <Typography style={{ paddingLeft: 10 }}>
+              {data.cronSyntax === ''
+                ? t(
+                    'chaosWorkflows.browseAnalytics.workFlowComparisonTable.once'
+                  )
+                : cronstrue.toString(data.cronSyntax)}{' '}
```

Comment:
please remove {' '}

Reply:
I have copied the code from schedules table and space is needed here for readability. @oumkale

Reply:
@ishangupta-ds  This can be removed, can you make this change in the schedule's page as well?

Reply:
Okay, then I am changing that file too. @amityt